modules/layers/src/arc-layer/arc-layer.ts

```diff
@@ -147,6 +148,15 @@ export default class ArcLayer<DataT = any, ExtraPropsT extends {} = {}> extends
     model?: Model;
   };
 
+  getBounds(): [number[], number[]] | null {
+    const attributeManager = this.getAttributeManager();
+    if (!attributeManager) return null;
+    const {instanceSourcePositions, instanceTargetPositions} = attributeManager.attributes;
+    const sourceBounds = instanceSourcePositions.getBounds();
+    const targetBounds = instanceTargetPositions.getBounds();
+    return sourceBounds && targetBounds && mergeBounds(sourceBounds, targetBounds);
```

Comment:
An issue here is `getBounds` for most layers are cached, meaning you can shallow compare two bounds to tell if it has changed. I think some of the extensions are implemented with this assumption.